spec/integration/master_process_spec.rb

```diff
@@ -104,7 +104,7 @@ def stub_quality_metrics_response
       passive: { 'count' => 5 },
       repeated_words: { 'count' => 8 },
     }
-    stub_request(:post, 'https://gov-quality-metrics.herokuapp.com/metrics').
+    stub_request(:post, 'https://govuk-content-quality-metrics.cloudapps.digital/metrics').
```

Comment:
Could you please fix up this commit with the previous one? In theory all commits should have all tests passing.